lib/dockerfile/ioncube-loader.m4

```diff
@@ -0,0 +1,5 @@
+RUN curl http://downloads3.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz -o /ioncube_loaders_lin_x86-64.tar.gz && \
+    tar -xzvf /ioncube_loaders_lin_x86-64.tar.gz -C / && \
+    cp "/ioncube/ioncube_loader_lin_${ION_CUBE_PHP_VERSION}.so" $PHP_EXTENSION_DIR && \
+    echo "zend_extension=${PHP_EXTENSION_DIR}/ioncube_loader_lin_${ION_CUBE_PHP_VERSION}.so" > /usr/local/etc/php/conf.d/00-ioncube.ini && \
```

Comment:
Using `docker-php-ext-enable` resulted in the extension not being loaded as a zend_extension, which threw errors. Couldn't see a way using that command to load a `zend_extension`